src/styles/color.js

```diff
@@ -30,8 +30,8 @@ export default ({ a11yColors = false }) => ({
 
   // FUNCTIONAL COLOURS FOR PRODUCT DEVELOPMENT
 
-  cta: a11yColors ? '#2D67FF' : '#0046FF',
-  positive: a11yColors ? '#008A00' : '#00D200',
-  negative: a11yColors ? '#E81700' : '#FF1900',
-  index: a11yColors ? '#FFCF00' : '#C15700',
+  cta: a11yColors ? '#0046FF' : '#2D67FF',
```

Comment:
think these still are wrong
0046FF, 00D200, FF1900 .   are normal colors .  not a11y colors